services/src/main/java/org/keycloak/forms/login/freemarker/model/WebAuthnAuthenticatorsBean.java

```diff
@@ -37,7 +37,7 @@ public WebAuthnAuthenticatorsBean(KeycloakSession session, RealmModel realm, Use
                 .map(webAuthnCredential -> {
                     String credentialId = Base64Url.encodeBase64ToBase64Url(webAuthnCredential.getWebAuthnCredentialData().getCredentialId());
                     String label = (webAuthnCredential.getUserLabel() == null || webAuthnCredential.getUserLabel().isEmpty()) ? "label missing" : webAuthnCredential.getUserLabel();
-                    String createdAt = DateTimeFormatterUtil.getDateTimeFromMillis(webAuthnCredential.getCreatedDate(), realm.getDefaultLocale());
+                    String createdAt = DateTimeFormatterUtil.getDateTimeFromMillis(webAuthnCredential.getCreatedDate(), session.getContext().resolveLocale(user));
```

Comment:
@mposolda The default provider for `LocaleSelectorProvider` properly handles situation, when the user is null. IMO, everybody, who creates the custom provider bears in mind the user can be null. Therefore, I didn't explicitly distinguish whether the user is present or not.